samcli/lib/remote_invoke/exceptions.py

```diff
@@ -0,0 +1,11 @@
+"""
+Exceptions used by remote invoke executors
+"""
+
+
+class InvalidResourceBotoParameterException(Exception):
```

Comment:
If this is catch in some other place then we can extend it from `Exception`. But if this will be thrown back to user, then it should extend from `UserException` which is handled by click and execution is ended gracefully (without printing stack trace)